moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.h

```diff
@@ -150,12 +150,15 @@ private Q_SLOTS:
   void approximateIKChanged(int state);
 
   // Planning tab
+  void computeCartecianPlanButtonClicked();
+  void computeJointSpacePlanButtonClicked();
```

Comment:
These two new functions are called ButtonClicked(), implying that they are connected to a button, but they are not. I would remove the "ButtonClicked" from the function name.

On second thought, is that file automatically generated by Qt? Should it be modified like this at all? See the comments [at the bottom of the file](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.h#L321).

Reply:
no in is hand written. so removed  from function name.